pages/common/gh-pr-merge.md

```diff
@@ -0,0 +1,32 @@
+# gh pr merge
+
+> Merge a pull request on GitHub.
+> More information: <https://cli.github.com/manual/gh_pr_merge>.
+
+- Merge the pull request associated with the current branch, using the default merge strategy:
+
+`gh pr merge`
+
+- Merge a specific pull request:
```

Comment:
```suggestion
- Merge the specified pull request using the default merge strategy:
```

Reply:
I make a mistake here, why `the specified`? we use `a specific` :thinking: 
```suggestion
- Merge a specific pull request interactively:
```

Reply:
@navarroaxel we do seem to use `a specific` more than `the specified` (507 files vs. 149 files), but in this case, the specific sounds better to me.  I'd prefer that we use it.

Reply:
so, I leave it as-is? what about your another comment?

Reply:
@bl-ue so, I leave it as-is? what about your another comment?

Reply:
So is "using the default merge strategy" not correct?  Do you specify the merge strategy interactively?

Reply:
Yeah the merge strategy, the body message and the branch deletion (if possible) it's interactively

Reply:
@bl-ue Yeah, the merge strategy, the body message and the branch deletion (if possible) it's interactively

Reply:
Then how about:
```suggestion
- Merge the specified pull request interactively:
```

Reply:
@bl-ue applied in af7bcab82a2cea89b571a14c6bc52323248ec996